src/views/Home.vue

```diff
@@ -1,8 +1,17 @@
 <template>
   <div id="home">
     <hero></hero>
-
+    <h2>
+      Listen to the assholes ramble
+    </h2>
     <sound-cloud></sound-cloud>
+  <h2>
+    You sing and you sing good!
+  </h2>
+    <h3>
+      (on video)
+    </h3>
+    <youtube></youtube>
```

Comment:
👍

Reply:
were you able to find it or is the component not there? I may have forgotten to add those like a fool haha